pkg/cri/server/metrics.go

```diff
@@ -34,6 +34,11 @@ var (
 	containerCreateTimer metrics.LabeledTimer
 	containerStopTimer   metrics.LabeledTimer
 	containerStartTimer  metrics.LabeledTimer
+
+	imagePullingError     metrics.LabeledCounter
```

Comment:
Should we use a latency metric instead, with `error`, `registry` as labels? By doing that users will be able to 

1. see the image pull latencies,
2. see both error count and success count, and therefore calculate the error ratio.

Reply:
To add a count for both error pulling and success pulling, I agree.
```
	imagePulls = ns.NewLabeledCounter("image_pulling", "count of image pulling by registry", "error", "registry")
```

For image pull latencies, do you mean the pulling duration? Could you explain? 
- for the duration, I think imagePullThroughput would be better as the image size would 1MiB or 1GB. The latency does not make sense in some scenarios.

Reply:
If we add the error as a label, some error message contains the image name and some detailed information. If so, the registry label is not that important. If not, we have to make the error more groupable.

Reply:
I was thinking of making this metric something like this:

```
imagePullTimer = ns.NewLabeledTimer("image_pull", "time to pull an image", "error", "registry")
```
, where error is just a boolean indicating whether the pull succeeds or not.

By doing this:
1. if user wants to get the overall pulling duration, regardless of each image size, they can get this metric with `error=='false'`.
2. if user wants to get the number of failed image pulls, they can get counts of this metric with `error=='true'` 


But yeah, I think what you are doing here (i.e. separating the counter metric from duration/throughput metric) is also fine. I don't have a strong preference here.

Reply:
I added like 
```
	imagePulls = ns.NewLabeledCounter("image_pulls", "count of image pulling by registry", "status", "registry")
```

Two value is valid: `status=failed` and `status=succeed`